samples/plugins/mysql_odbc/connectionResolver.tdr

```diff
@@ -0,0 +1,24 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<tdr class='mysql_odbc'>
+	<connection-resolver>
+    <connection-builder>
+      <script file='connectionBuilder.js'/>
+    </connection-builder>
+    <connection-normalizer>
+      <required-attributes>
+        <attribute-list>
```

Comment:
I think you're missing some attributes here.

Reply:
Think I've added all that are appropriate, appreciate you taking a look again.

Reply:
Looks good!